Source/Extensions/Blazorise.Charts/BaseChartOfT.cs

```diff
@@ -86,6 +93,11 @@ public void NotifyPluginInitialized( string pluginName )
             pluginNames.Add( pluginName );
         }
     }
+    
+    public void NotifyBasePluginInitialized( BaseChartPlugin plugin )
```

Comment:
Do we still need `NotifyBasePluginInitialized`? Can we merge it with `NotifyPluginInitialized`?

Reply:
When proceeding with the update, there will be just the `NotifyPluginInitialized( BaseChartPlugin plugin )`, this is just to keep the both "systems" running.